### PR TITLE
Fixes spies having multi-use autosurgeons

### LINKED ISSUE #81827: The Nuclear Operative cybernetic implants that spies can get come in the reusable autosurgeon variant
## Issue Summary

Let's say the Mr. Tern Coate managed to complete a bounty for a nice and shiny CNS Rebooter. Well, the Rebooter is nice and all but once Mr. Tern Coate has used up the CNS Rebooter, nothing actually stops them from filling up that autosurgeon with literally any organ they want and keep using it to inject all kinds of things into themselves. Like more cyberware from science.

This isn't the case for any implants that would normally be available to traitors and not operatives (like the Laser Arm implant). The reason operatives have a reusable variant is...well, who really cares if they procure more cyberware on the station, that's fine. It's more difficult and convoluted for them to do that. Spies are crew, they've got some degree of a cover. As a result, spies can mostly bypass the typical restriction of other players to gain an increase in potential cybernetic prowess.

## Reproduction:

It's not really worth reproducing some steps here, it should be self explanatory.

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Fixes https://github.com/tgstation/tgstation/issues/81827

## Why It's Good For The Game

Autosurgeons are deliberately not resuable for almost all instances they show up in the game. 

This is fine for a single use insertion of a highly illegal organ, the autosurgeon then allowing you to entirely ignore surgery and player interaction to get various power boosts via cybernetics from that point onwards seems unintended.

## Changelog
:cl:
fix: Spies no longer have access to infinite use autosurgeons.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items/nukeops.dm [hunk 1]
```diff
@@ -744,19 +744,17 @@
 	cost = 6
 	purchasable_from = UPLINK_NUKE_OPS | UPLINK_SPY
 
-/datum/uplink_item/implants/nuclear/reviverplus
+/datum/uplink_item/implants/nuclear/reviver
```
Comment (nevimer) [inline code/modules/uplink/uplink_items/nukeops.dm:747]: You renamed it to reviver here, taking away the name reviverplus

File: code/modules/uplink/uplink_items/spy_unique.dm [hunk 2]
```diff
@@ -121,3 +121,18 @@
 	name = "Syndicate First Medic Kit"
 	desc = "A syndicate tactical combat medkit, but only stocked enough to do basic first aid."
 	item = /obj/item/storage/medkit/tactical_lite
+
+/datum/uplink_item/implants/spy_unique/antistun
+	name = "CNS Rebooter Implant"
+	desc = "This implant will help you get back up on your feet faster after being stunned. Comes with an autosurgeon."
+	item = /obj/item/autosurgeon/syndicate/anti_stun/single_use
+
+/datum/uplink_item/implants/spy_unique/reviverplus
```
Comment (nevimer) [inline code/modules/uplink/uplink_items/spy_unique.dm:130]: But you use the name reviverplus here? Won't it get confusing?

Comment (necromanceranne) [reply code/modules/uplink/uplink_items/spy_unique.dm:130]: copypaste moment

**Comment by Admiral-Gigaton** [comment]
Dammit arcane, I didn't even get to try it out and become inspector gadget!

File: code/modules/uplink/uplink_items/spy_unique.dm [hunk 3]
```diff
@@ -121,3 +121,18 @@
 	name = "Syndicate First Medic Kit"
 	desc = "A syndicate tactical combat medkit, but only stocked enough to do basic first aid."
 	item = /obj/item/storage/medkit/tactical_lite
+
+/datum/uplink_item/implants/spy_unique/antistun
+	name = "CNS Rebooter Implant"
+	desc = "This implant will help you get back up on your feet faster after being stunned. Comes with an autosurgeon."
```
[the remark this answers is not in the document]
Comment (necromanceranne) [reply code/modules/uplink/uplink_items/spy_unique.dm:127]: I yeah I remember I think you mentioning this is a thing, sounds good to me

**Pull request merged**